include/NovelRT/Audio/Audio.h

```diff
@@ -0,0 +1,33 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_AUDIO_H
+#define NOVELRT_AUDIO_H
+
+// Dependencies for Audio
+#include "../Exceptions/Exceptions.h"
+#include "../LoggingService.h"
+#include "../Utilities/Lazy.h"
+#include "../Utilities/Misc.h"
+#include <AL/al.h>
+#include <AL/alc.h>
+#include <cstdint>
+#include <sndfile.h>
+#include <vector>
```

Comment:
This won't compile if in C API land. Not sure how this is currently passing. I think we need to review the structure of your C API flag here, because I'm becoming more and more confused the more I look at this file. 😅

Reply:
This shouldn't need to be compiled in C API land - it's a C++ header 😄 

The call that you see afterwards is only if c++ is defined and not NOVELRT_C_API - this is only used for C# generation AFAIK. C API will strictly use the Interop headers (where this doesn't exist).
I'll still play around with it, but removing the section that used to exist for C# generation might cause it to break due to the underlying types being typedef'ed

Reply:
Yeah, its odd but NovelRT.h seems to handle most of the C_API flag stuff, so I've just removed it. That way, if we're in C API, this isn't being compiled but the Interop Header is.